.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+# https://github.com/actions/labeler#common-examples
+# Generated from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cudf/labels
+
+cuDF (Python):
+  - 'python/**'
+  - 'notebooks/**'
+  
+cpp:
```

Comment:
@kkraus14 should this label be `libcudf` or `libcudf++`? Or do you prefer a new label here of `cpp`? Just trying to match to an [existing label](https://github.com/rapidsai/cudf/labels) and `cpp` is not one currently

Reply:
`libcudf` if we want to match an existing label